plugins/importexport/native/filter/NativeXmlSubmissionFileFilter.inc.php

```diff
@@ -276,8 +279,32 @@ function handleRevisionChildElement($node, $submission, $submissionFile) {
 				break;
 			case 'href':
 				$submissionFile->setFileType($node->getAttribute('mime_type'));
+				import('lib.pkp.classes.file.TemporaryFileManager');
+				$temporaryFileManager = new TemporaryFileManager();
+				$temporaryFilename = tempnam($temporaryFileManager->getBasePath(), 'src');
 				// Allow wrappers to handle URLs
-				return $node->getAttribute('src');
+				$filesrc = $node->getAttribute('src');
+				if (preg_match('|\w+://.+|', $filesrc)) {
+					import('lib.pkp.classes.file.FileWrapper');
+					$wrapper = FileWrapper::wrapper($filesrc);
+					file_put_contents($temporaryFilename, $wrapper->contents());
+					if (!filesize($temporaryFilename)) {
+						$deployment->addError(ASSOC_TYPE_SUBMISSION, $submission->getId(), __('plugins.importexport.common.export.error.inputFileNotReadable', array('param' => $filesrc)));
+					}
+					return $temporaryFilename;
+				}
+				// local file (absolute path)
+				if (substr($filesrc, 1, 1) === '/') {
```

Comment:
Should here maybe also be checked `if` `is_readable` (like in the next if statement?